src/scene/Scale.js

```diff
@@ -171,6 +173,13 @@ function ordinal(scale, rng, group) {
   if (!scale.invert && spatial) invertOrdinal(scale);
 }
 
+function setPadding(pad, config, defaultPad, parent_id, datum) {
+  if (!parent_id && config && config.scale && config.scale[pad] && !datum._facetID) {
+    return config.scale[pad];
+  }
+  return defaultPad;
+}
+
```

Comment:
@emilygu: If I understand this, the test for `parent_id` and `datum._facetID` ensures that a theme's scale paddings are only applied to the scales defined at the top-level (i.e., not nested within groups). Is that correct? If so, was this a design pattern you frequently saw? If yes, then I would say we want to break these out to separate padding variables. It should still be possible to set the `padding` and `outerPadding` for _all_ scales in a theme.